packages/server/src/async/consumers/consumer.js

```diff
@@ -0,0 +1,30 @@
+const assert = require('assert');
+const _ = require('lodash')
+
+const data = require('../../data');
+
+const { TEMPLATE_NAME } = require('../../models/enums/template-name');
+const { NOTIFICATION_TYPE } = require('../../models/enums/notification-type');
+const { EmailEnvelope } = require('../../models/classes/notifications');
+
+class Consumer {
+  constructor(options = {}) {
+    assert(_.isObject(options), 'Options is not an object');
```

Comment:
Why this pattern of using assert outside a test? I’m not used to that.

Reply:
Assertion is not only about tests. Assertion is useful to prevent development and runtime errors.